monitoring/storage.go

```diff
@@ -50,17 +81,29 @@ type HighWatermarkCheckerData struct {
 	AvailableBytes uint64 `json:"available_bytes"`
 }
 
-// FailureMessage returns failure watermark check message
-func (d HighWatermarkCheckerData) FailureMessage() string {
+// WarningMessage returns warning watermark check message
+func (d HighWatermarkCheckerData) WarningMessage() string {
+	return fmt.Sprintf("disk utilization on %s exceeds %v percent (%s is available out of %s), see https://gravitational.com/telekube/docs/cluster/#garbage-collection",
```

Comment:
I think this link needs updating as there's no more telekube: https://gravitational.com/telekube/docs/cluster/#garbage-collection.

The current link is https://gravitational.com/gravity/docs/cluster/#garbage-collection. Same below.